db/migrate/20220705162118_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] Your posts table uses `user_id`, whereas you have `author_id`. Kindy fix that by referencing `author`, instead of `user`. You can use the `to_table` option to refer Rails to the users' table.

  You can visit this answer here on how to reference a different table using a specific column in Rails: [https://stackoverflow.com/a/42056089](https://stackoverflow.com/a/42056089)

Reply:
Thank you @julie-ify 
I have updated it to author

Reply:
Thank you @julie-ify 
I have updated it to author